README.md

```diff
@@ -1,8 +1,10 @@
 # Terraform Provider
 
-- Website: https://www.terraform.io
-- [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
-- Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
+Website: https://www.terraform.io
+Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
+
+[![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
+[![Slack](https://cdn.brandfolder.io/5H442O3W/as/pl546j-7le8zk-5guop3/Slack_RGB.png?height=30)](https://kubernetes.slack.com/messages/CJY6ATQH4)
```

Comment:
I think this wants a registration link for the Slack too? (see [how Azure's readme does this](https://github.com/terraform-providers/terraform-provider-azurerm))

Reply:
Good tip! Let me add that.